### DISCUSSION TITLE
twinBASIC BETA 66

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#956] [Releases]
- improved: better error handling around quarantined IDE executable files [ https://github.com/WaynePhillipsEA/twinbasic/issues/929 ]

<hr /><em>This discussion was created from the release <a href='https://github.com/WaynePhillipsEA/twinbasic/releases/tag/beta-66'>twinBASIC BETA 66</a>.</em>

**Comment by mlcoq** [comment]
Maybe remove the malware

**Reply by WaynePhillipsEA** [reply]
There's no malware.  It's called false positives, which are very common with new releases of any software.

**Reply by mlcoq** [reply]
Okay, I withdraw my message

**Reply by mburns08109** [reply]
Thanks @WaynePhillipsEA - I will check it out tonight and see what/if-anything remains amiss.
